R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -2070,6 +2072,16 @@ test_that("dapply() on a DataFrame", {
   rownames(expected) <- NULL
   expect_identical(expected, result)
 
+  result <- dapplyCollect(
+              df,
+              function(x) {
+                y <- x[x$a > 1, ]
+                y <- cbind(y, y$a + 1L)
```

Comment:
Well, let's assume I use dapply to train linear model. The model object contains a list of different attributes for example coefficients, residuals , ... etc. 
model <- lm(Species ~ ., iris).
dapply wraps my model in a data.frame which is fine.

Now, as next I'd like to make predictions using dapply and the model which I've previously trained.
This means, that I probably need something like this: 
```
df1 <- dapply(
    df, 
   model,
    function (x, model) {
       predict (model, x, ... )
   }
)
```
Can we do something similar ?